src/AspNetCoreModuleV2/CommonLib/requesthandler.h

```diff
@@ -30,7 +38,26 @@ class REQUEST_HANDLER: public virtual IREQUEST_HANDLER
         }
     }
 
-    REQUEST_NOTIFICATION_STATUS OnAsyncCompletion(DWORD cbCompletion, HRESULT hrCompletionStatus) override
+
+    REQUEST_NOTIFICATION_STATUS
+    OnExecuteRequestHandler() final
```

Comment:
I think this is safe, but let's make sure the namechange is forwards and backwards compatible.